README.md

```diff
@@ -78,7 +74,7 @@ java -jar ui/target/ui.jar
 <img src="ui/src/main/resources/ui.jpg">
 </p>
 
-Dartagnan supports programs written in the `.c`, `.litmus` or `.bpl` (Boogie code generated by smack) formats.
+Dartagnan supports programs written in the `.c`, `.litmus` formats.
```

Comment:
I don't want to claim we support arbitrary llvm code. If someone compiles C++ or OpenCL code, I am pretty sure we will have issues.

Reply:
Sure, but we could still say that we support LLVM code generated from C. Not sure if this makes that much of a difference, but technically it would allow people to run their own LLVM passes before running Dartagnan. Maybe that is helpful in some processing pipeline?

Reply:
I don't think such things should go to a README. Sure, something like that works and it is what we do with libvsync, but it is not how we expect the normal user to interact with the tool.